kubernetes/travis/deploy-nextgen.sh

```diff
@@ -2,7 +2,7 @@
 
 export NEXTGEN_DEPLOY_BRANCH=${NEXTGEN_DEPLOY_BRANCH:-nextgen}
```

Comment:
Can you please change it to "development". Make sure there is no "nextgen" in this file.

Reply:
Should I remove all occurrences of nextgen ?
Or just this file ?

Reply:
nextgen in readme file is taken care in an existing PR. You can update any other file.

Reply:
Ok, just updated the file that you said. Checking for others

Reply:
What about the file name ?